.github/workflows/checks.yml

```diff
@@ -12,8 +12,9 @@ jobs:
     name: Download deps, install poetry
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
Does it mean that if one test fails the others will continue running?

Reply:
[Yes](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast)

Reply:
But I don't think we need the matrix anymore, as only python3.9 is supported

Reply:
I wouldn't include it if we are using only `3.9` for now anyways. I'd address changing the checks in the separate issue.

Reply:
`fail-fast` only applies to a strategy for *matrix* anyway. If set to true, the same jobs (but for different items from matrix) fail, once any job fails.